src/interactions/button/verifyRulesButton.ts

```diff
@@ -0,0 +1,31 @@
+import { ButtonInteraction, MessageActionRow, Modal, ModalActionRowComponent, TextInputComponent } from 'discord.js';
+import { ButtonInteractionHandler } from './buttonInteractionHandler';
+import { Cooldown } from '#util/cooldown';
+import Client from '#src/Client';
+
+export class VerifyRulesButton implements ButtonInteractionHandler {
+    readonly client: Client;
+    readonly customId = 'verifyRules';
+    readonly cooldown: Cooldown;
+
+    constructor(client: Client) {
+        this.client = client;
+        this.cooldown = new Cooldown(60, 3);
```

Comment:
We can set a shorter cooldown for this, since showing a modal isn't very expensive and users could accidentally exit the modal. Perhaps something like 3 in 5 seconds.